x-pack/legacy/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/categorization_view/examples_valid_callout.tsx

```diff
@@ -64,13 +67,16 @@ export const ExamplesValidCallout: FC<Props> = ({ examplesValid, categorizationA
   );
 };
 
-const PercentageText: FC<{ examplesValid: number }> = ({ examplesValid }) => (
+const PercentageText: FC<{ examplesValid: number; sampleSize: number }> = ({
+  examplesValid,
+  sampleSize,
+}) => (
   <div>
     <FormattedMessage
       id="xpack.ml.newJob.wizard.pickFieldsStep.categorizationFieldPercentage"
       defaultMessage="{number} field values analyzed, {percentage}% contain valid tokens."
       values={{
-        number: NUMBER_OF_CATEGORY_EXAMPLES * CATEGORY_EXAMPLES_MULTIPLIER,
+        number: sampleSize,
```

Comment:
Not introduced directly by this PR, but the `values` part of the text should switch to `value` if only one value. e.g. gallery data, select `action`:

![image](https://user-images.githubusercontent.com/7405507/72356001-6303f780-36e0-11ea-92a4-249496559c1a.png)